internal/pkg/aws/regions/regions.go

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package regions
+
+import (
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/copilot-cli/internal/pkg/aws/partitions"
```

Comment:
This package is dependent on `internal/pkg/aws/partitions` - I think ideally the dependencies of `internal/pkg/aws` should be external package only.  Moreover, `partitions` package also rely on `aws/endpoints` package. So I think it probably makes sense to merge this `regions` package with `partitions` package